lib/src/api/engine/remote/http/mod.rs

```diff
@@ -27,8 +28,8 @@ use crate::api::Surreal;
 use crate::dbs::Status;
 use crate::headers::AUTH_DB;
 use crate::headers::AUTH_NS;
-use crate::headers::DB;
-use crate::headers::NS;
+use crate::headers::DB_LEGACY;
+use crate::headers::NS_LEGACY;
```

Comment:
why the renames?

Reply:
The new names break backwards compatibility. `v1.0` servers don't understand the new header names so the namespaces and databases passed are ignored. The causes the SDK to not be able to talk to v1.0 servers.